lib/thredded.rb

```diff
@@ -24,30 +24,31 @@
 require 'thredded/engine'
 
 module Thredded
-  mattr_accessor :user_class,
-    :user_name_column,
+  mattr_accessor \
+    :active_user_threshold,
     :avatar_url,
-    :email_incoming_host,
     :email_from,
+    :email_incoming_host,
     :email_outgoing_prefix,
     :email_reply_to,
-    :user_path,
     :layout,
-    :active_user_threshold
+    :user_class,
+    :user_name_column,
+    :user_path
 
   # @return [Symbol] The name of the moderator flag column on the users table for the default permissions model
   mattr_accessor :moderator_column
 
   # @return [Symbol] The name of the admin flag column on the users table for the default permissions model
   mattr_accessor :admin_column
 
-  self.user_name_column = :name
-  self.avatar_url = ->(user) { Gravatar.src(user.email, 128, 'mm') }
   self.active_user_threshold = 5.minutes
-  self.layout = 'thredded/application'
+  self.admin_column = :admin
+  self.avatar_url = ->(user) { Gravatar.src(user.email, 128, 'mm') }
   self.email_reply_to = -> postable { "#{postable.hash_id}@#{Thredded.email_incoming_host}" }
+  self.layout = 'thredded/application'
   self.moderator_column = :admin
-  self.admin_column = :admin
+  self.user_name_column = :name
```

Comment:
These changes are merely for formatting and alphabetical ordering.

... Because OCD.